packages/components-native/src/InputText/context/InputAccessoriesProvider.style.tsx

```diff
@@ -14,7 +14,7 @@ export const styles = StyleSheet.create({
     height: BAR_HEIGHT,
   },
   lightTheme: {
-    backgroundColor: tokens["color-surface--background"],
+    backgroundColor: tokens["color-interactive--background"],
```

Comment:
```suggestion
    backgroundColor: tokens["color-surface--background"],
```

This is the toolbar above the native keyboard for things like the "Done" button, this isn't a "UI element" in the sense of the other components here